src/Uno.Sdk/packages.json

```diff
@@ -292,6 +292,7 @@
 		"Uno.Extensions.Hosting.WinUI",
 		"Uno.Extensions.Http.WinUI",
 		"Uno.Extensions.Http.Refit",
+		"Uno.Extensions.Http.Kiota",
```

Comment:
Not sure if this part was needed to change?

Reply:
Yes it's needed, it's also going to be needed on the template repo as well.